src/index.css

```diff
@@ -92,3 +92,13 @@ code {
 ul {
 	padding: 0;
 }
+
+.add-prompt {
+	color: var(--color-white);
+	background-color: #efefef;
+	border-radius: 5px;
+	border: 1px solid black;
+	padding: 4px 6px;
+	text-decoration: none;
+	color: black;
```

Comment:
If you are going to use the provided color variables, then maybe here you should also use `var(--color-black);` :slightly_smiling_face:

Reply:
Fixed this, thanks!